processing/src/main/java/io/druid/query/expression/TimestampParseExprMacro.java

```diff
@@ -56,9 +56,11 @@ public Expr apply(final List<Expr> args)
       timeZone = DateTimeZone.UTC;
     }
 
-    final DateTimeFormatter formatter = formatString == null
-                                        ? ISODateTimeFormat.dateTimeParser()
-                                        : DateTimeFormat.forPattern(formatString).withZone(timeZone);
+    final DateTimes.UtcFormatter formatter = DateTimes.wrapFormatter(
```

Comment:
`DateTimes.wrapFormatter(ISODateTimeFormat.dateTimeParser())` could be cached more globally. Maybe create a constant in DateTimes.

Reply:
:+1: